authors/apps/articles/extra_methods.py

```diff
@@ -0,0 +1,18 @@
+import random
+import string
+from django.utils.text import slugify
+from .models import Article
+
+def random_string_generator(size=10, chars=string.ascii_lowercase + string.digits):
+    return ''.join(random.choice(chars) for _ in range(size))
+
+def create_slug(object_model, title):
```

Comment:
Well done @AtamaZack, It's good you have created the "create_slug" method, now we call also use if it in our tests to create a slug for an article